//  Error handling Example using try-catch block
let a = 10;
let b = 20;

console.log(a + b);
console.log(a - b);
try{
    console.log(a * c);
}catch(err)
{
    console.log(err);
}
//  It ia prints after adding try-catch block
console.log(a % b);
console.log(a / b);
console.log(a + b);
console.log(a - b);
console.log(a * b);